automod/commands.go

```diff
@@ -198,14 +198,18 @@ func (p *Plugin) AddCommands() {
 			}
 
 			for name, count := range violations {
-				out += fmt.Sprintf("Violation: %-20s Count: %d\n", name, count)
+				if len([]rune(name)) > 31 {
+					out += fmt.Sprintf("%-30s_ Count: %d\n", string([]rune(name)[:30]), count)
+				} else {
+					out += fmt.Sprintf("%-31s Count: %d\n", name, count)
+				}
```

Comment:
Consider using `CutStringShort`. Note that it does not have precisely the same effect -- it uses `...` to mark truncated text as opposed to `_` -- but IMO this is not a major issue; violation names should generally be short.

Reply:
I am in favour of this too, "..." is generally a globally understood term for a truncated text.

Reply:
changes reflected in 51aec6e